module/Database/view/database/query/export.phtml

```diff
@@ -41,6 +41,6 @@ rewind($mem);
 
 // send correct download headers
 header('Content-Type: application/csv');
-header('Content-Disposition: attachement; filename="query.csv"');
+header('Content-Disposition: attachement; filename="' . $name . '.csv"');
```

Comment:
Potential escape necessary?

Reply:
![{6DB7BAE6-F3DB-49DF-94BB-C2A3475D54A9}](https://github.com/user-attachments/assets/a0add5fe-c175-435d-b7b6-3da0c0412cd9)
`escapeHtml` results in this